conf/app.ini

```diff
@@ -184,6 +184,8 @@ SSL_MODE = disable
 PATH = data/gitea.db
 ; For "sqlite3" only. Query timeout
 SQLITE_TIMEOUT = 500
+; For interate buffer
```

Comment:
Default setting should be mentioned too IMO.

Reply:
And it should be `For iterate buffer`

Reply:
@Morlinest the default is 50, this file is an example file.
@daviian done.

Reply:
IMHO it is good practice to comment defaults in config files (even in example files). If you search `default` in this file, you can find a lot of comments like `... (default false)` or `... default is ...`. If you look at file you should know what happens if you don't set some variables.

Reply:
@Morlinest done.